app/src/main/res/layout/activity_user.xml

```diff
@@ -54,31 +54,49 @@
 
                         <!--user name-->
                         <TextView
+                            android:id="@+id/textView"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
+                            android:text="@{viewModel.user.name}"
                             android:textColor="@color/colorPrimary"
                             android:textSize="@dimen/sp_20"/>
 
                         <!--user phone-->
                         <TextView
+                            android:text="@{viewModel.user.phone}"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
                             android:textSize="@dimen/sp_15"/>
 
                         <!--user email-->
                         <TextView
+                            android:textColor="@color/colorPrimary"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
                             android:text="@{viewModel.user.email}"
                             android:textSize="@dimen/sp_15"/>
 
-                        <!--display user's point-->
-                        <TextView
+                        <RelativeLayout
                             android:layout_width="match_parent"
-                            android:layout_height="wrap_content"
-                            android:gravity="center"
-                            android:textColor="@color/color_red"
-                            android:textSize="@dimen/sp_15"/>
+                            android:layout_height="wrap_content">
+                            <!--display user's point-->
+                            <TextView
+                                android:text="@{viewModel.user.birthday}"
+                                android:layout_width="wrap_content"
+                                android:layout_height="match_parent"
+                                android:textColor="@color/colorPrimary"
+                                android:textSize="@dimen/sp_15"/>
+                            <TextView
```

Comment:
WARNING: [RelativeOverlap] `TextView-5` can overlap `TextView-3` if @{viewModel.user.birthday}, @string/title_na grow due to localized text expansion
 If relative layout has text or button items aligned to left and right sides they can overlap each other due to localized text expansion unless they have mutual constraints like `toEndOf`/`toStartOf`.